rules/windows/process_creation/proc_creation_win_lolbin_cscript_gathernetworkinfo.yml

```diff
@@ -1,28 +0,0 @@
-title: GatherNetworkInfo.vbs Script Usage
-id: 575dce0c-8139-4e30-9295-1ee75969f7fe
-status: test
```

Comment:
obligatory `git mv` reminder :innocent:

Reply:
This one something weird happened XD, I did a git mv but then I saved it again mistakenly and I happen to have 2 of the same file with different names 😛 not my fault (git is hard)